pillar.example

```diff
@@ -34,6 +34,11 @@ postgresql:
     wal_keep_segments: (unset)
     wal_buffers: (unset)
 
+    log_statement: 'none'
+    logging_collector: off
+    log_directory: (unset) # NOTE: Will only be deployed
```

Comment:
The multi-line comment is confusing. Use the following:

```
log_statement:

# NOTE: ...
log_directory:
log_filename: 
```

Alternatively, use a single line comment and copy it to both lines.